src/typing/flow_js.ml

```diff
@@ -930,7 +953,8 @@ let rec __flow cx ((l: Type.t), (u: Type.use_t)) trace =
     (* The sink component of an annotation constrains values flowing
        into the annotated site. *)
 
-    | _, UseT (use_op, AnnotT (sink_t, _)) ->
+    | l, UseT (use_op, AnnotT (sink_t, _))
```

Comment:
This is part of the PR's "enforce subtyping on structural class statics" clause. It should probably be exported and rebased below this PR.